luarules/gadgets/map_lava.lua

```diff
@@ -25,13 +25,10 @@ if gadgetHandler:IsSyncedCode() then
 
 	local lavaLevel = lava.level
 	local lavaGrow = lava.grow
-	local lavaDamage = lava.damage
-	local lavaDamageMode = lava.damageMode
+
+	-- damage is specified in health lost per second, damage is applied every 10 frames
+	local lavaDamage = lava.damage/3.0
```

Comment:
Might want to do
```lua
local DAMAGE_RATE = 10 -- frames
local lavaDamage = lava.damage * (DAMAGE_RATE / Game.gameSpeed)
```
and then `if frame % DAMAGE_RATE == 0` somewhere below where the magic constant 10 currently lives

Reply:
done